test/metabase/lib/drill_thru_test.cljc

```diff
@@ -439,6 +441,43 @@
                     (lib/available-drill-thrus query -1 context)))
             (test-drill-applications query context)))))))
 
+(deftest ^:parallel table-view-available-drill-thrus-aggregate-column-header-test
+  (let [query (-> (lib/query meta/metadata-provider (meta/table-metadata :orders))
+                  (lib/aggregate (lib/count))
+                  (lib/aggregate (lib/sum (meta/field-metadata :orders :tax)))
+                  (lib/aggregate (lib/max (meta/field-metadata :orders :discount)))
+                  (lib/breakout (meta/field-metadata :orders :product-id))
+                  (lib/breakout (-> (meta/field-metadata :orders :created-at)
+                                    (lib/with-temporal-bucket :month))))]
+    (testing "Drills for count aggregation"
+      (let [count-col (m/find-first (fn [col]
+                                      (= (:display-name col) "Count"))
+                                    (lib/returned-columns query))]
+        (is (some? count-col))
+        (let [context {:column count-col
+                       :value  nil}]
+          (is (=? [{:type   :drill-thru/sort
+                    :column {:name "count"}}
+                   {:type         :drill-thru/summarize-column
+                    :column       {:name "count"}
+                    :aggregations [:distinct :sum :avg]}]
+                  (lib/available-drill-thrus query -1 context)))
+          (test-drill-applications query context))))
+    (testing "Drills for max(discount) aggregation"
+      (let [max-of-discount-col (m/find-first (fn [col]
+                                                (= (:display-name col) "Max of Discount"))
+                                              (lib/returned-columns query))]
+        (is (some? max-of-discount-col))
+        (let [context {:column max-of-discount-col
+                       :value  nil}]
+          (is (=? [{:type   :drill-thru/sort
+                    :column {:display-name "Max of Discount"}}
+                   {:type         :drill-thru/summarize-column
+                    :column       {:display-name "Max of Discount"}
+                    :aggregations [:distinct :sum :avg]}]
+                  (lib/available-drill-thrus query -1 context)))
+          (test-drill-applications query context))))))
```

Comment:
The rest of the test is a port of 

https://github.com/metabase/metabase/blob/5ab317540d064ac3d402508d0038629e8a041575/frontend/src/metabase-lib/drills.unit.spec.ts#L701